tracer/src/Datadog.Trace/ExtensionMethods/ArrayExtensions.cs

```diff
@@ -18,5 +18,11 @@ public static T[] Concat<T>(this T[] array, params T[] newElements)
 
             return destination;
         }
-    }
+
+        public static string PrintContents<T>(this T[] array)
```

Comment:
nit: I'm not a big fan of adding these sort of extension methods. They tend to only be used once IME, and could easily be inlined, but it's not a hill I'm willing to die on😉

Reply:
Agree with you, converted to local method in a9af0917c4fa07c264cc51c46052a37f6640aff4